control-plane/pkg/reconciler/base/reconciler.go

```diff
@@ -55,6 +57,10 @@ const (
 
 	Protobuf = "protobuf"
 	Json     = "json"
+
+	ReceiverPort = "8080"
```

Comment:
Maybe `ProbeReceiverPort`? Even if it's the same server, IMO it's good to clarify that we use it only for that purpose

Reply:
SGTM.